src/app/clusters/actions-server/actions-server.cpp

```diff
@@ -220,139 +186,90 @@ void ActionsServer::HandleCommand(HandlerContext & handlerContext, FuncT func)
-    Status status       = Status::InvalidInState;
-    if (delegate != nullptr)
-    {
-        status = delegate->HandleInstantAction(commandData.actionID, commandData.invokeID);
-    }
+    Status status = Status::InvalidInState;
+    status        = mDelegate->HandleInstantAction(commandData.actionID, commandData.invokeID);
     ctx.mCommandHandler.AddStatus(ctx.mRequestPath, status);
 }
 
 void ActionsServer::HandleInstantActionWithTransition(HandlerContext & ctx,
                                                       const Commands::InstantActionWithTransition::DecodableType & commandData)
 {
-    Delegate * delegate = GetDelegate(ctx.mRequestPath.mEndpointId);
-    Status status       = Status::InvalidInState;
-    if (delegate != nullptr)
-    {
-        status =
-            delegate->HandleInstantActionWithTransition(commandData.actionID, commandData.transitionTime, commandData.invokeID);
-    }
+    Status status = Status::InvalidInState;
+    status = mDelegate->HandleInstantActionWithTransition(commandData.actionID, commandData.transitionTime, commandData.invokeID);
     ctx.mCommandHandler.AddStatus(ctx.mRequestPath, status);
 }
 
 void ActionsServer::HandleStartAction(HandlerContext & ctx, const Commands::StartAction::DecodableType & commandData)
 {
-    Delegate * delegate = GetDelegate(ctx.mRequestPath.mEndpointId);
-    Status status       = Status::InvalidInState;
-    if (delegate != nullptr)
-    {
-        status = delegate->HandleStartAction(commandData.actionID, commandData.invokeID);
-    }
+    Status status = Status::InvalidInState;
+    status        = mDelegate->HandleStartAction(commandData.actionID, commandData.invokeID);
     ctx.mCommandHandler.AddStatus(ctx.mRequestPath, status);
 }
 
 void ActionsServer::HandleStartActionWithDuration(HandlerContext & ctx,
                                                   const Commands::StartActionWithDuration::DecodableType & commandData)
 {
-    Delegate * delegate = GetDelegate(ctx.mRequestPath.mEndpointId);
-    Status status       = Status::InvalidInState;
-    if (delegate != nullptr)
-    {
-        status = delegate->HandleStartActionWithDuration(commandData.actionID, commandData.duration, commandData.invokeID);
-    }
+    Status status = Status::InvalidInState;
+    status        = mDelegate->HandleStartActionWithDuration(commandData.actionID, commandData.duration, commandData.invokeID);
     ctx.mCommandHandler.AddStatus(ctx.mRequestPath, status);
 }
 
 void ActionsServer::HandleStopAction(HandlerContext & ctx, const Commands::StopAction::DecodableType & commandData)
 {
-    Delegate * delegate = GetDelegate(ctx.mRequestPath.mEndpointId);
-    Status status       = Status::InvalidInState;
-    if (delegate != nullptr)
-    {
-        status = delegate->HandleStopAction(commandData.actionID, commandData.invokeID);
-    }
+    Status status = Status::InvalidInState;
+    status        = mDelegate->HandleStopAction(commandData.actionID, commandData.invokeID);
```

Comment:
I'll stop commenting on this.  Just fix them all, please.

Reply:
Fixed at all places.